src/components/Timestamp.tsx

```diff
@@ -4,24 +4,33 @@ import React from 'react';
 interface TimestampProps {
   timestamp: Scalars['DateTimeOffset'];
 }
+
+const formatOptions = {
```

Comment:
How about using with `Intl.DateTimeFormatOptions`?
```suggestion
const formatOptions: Intl.DateTimeFormatOptions = {
```